src/en/federation/federation.md

```diff
@@ -13,4 +13,31 @@ One way you can take advantage of federation is by opening a different instance,
 - `https://lemmy.ml/post/123` (Post)
 - `https://lemmy.ml/comment/321` (Comment)
 
-You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
\ No newline at end of file
+You can see the list of linked instances by following the "Instances" link at the bottom of any Lemmy page.
+
+## Searching for communities
+
+If you search for a community first time, 20 posts are fetched initially. Only if a least one user on your instance subscibes this community on the other host, this host will deliver updates to your instance.
+Updates mean: 
+- new posts
+- new comments
+- votings
+- edited comments and posts
+
+You can copy the URL of the community from the address-bar in your browser and insert it in your search-field. Wait a few seconds, the post will appear below. There is no indicator for a running search now!
+
+## Searching for posts
+
+Copy the URL of a post in your lemmy's search-field. Wait a few seconds until the post will appear. Klick on it, you'll be lead to the posting into the community, where you cann see the "Follow"-button and follow the community. Be aware, not all postings are fetched from the community, only a few ammount, and no comments!
+
+## Searching for comments
+
+If you find an interesting comment under a posting on another instance, you can find below the comment in the 3-dot-menu the link-symbol. Copy this link, oder klick on the link-button and copy the url from your address-bar in the browser. The link looks like `https://lemmy.ml/post/56382/comment/40796`. Remove the `post/XXX` part from this link and put it into your search-bar. For this example, search for `https://lemmy.ml/comment/40796`. 
+This comment, all parent comments, users and community and the responding posting is fetched from the remote instance, if they are not known locally.
+No siblings are fetched!!
+
+If you wand more comments from older postings, you have to search for them the same way as described just above.
+
```

Comment:
Users can also be fetched by URL, but that currently doesnt fetch any further data.